.gitignore

```diff
@@ -1 +1,3 @@
 **/.kitchen/
+**/terraform.*
+**/.terraform/*
```

Comment:
Terraform generates a lot of state and stores that locally so these will ignore those files and prevent them from being checked in.